packages/debian/postinst

```diff
@@ -120,7 +119,7 @@ register_adu_user_and_process_with_eis() {
         if [ -d "$eis_idservice_dir" ]; then
 
             if [ ! -f "$eis_idservice_dir/$adu_eis_conf_file" ]; then
-                printf "[[principal]]\n name=\"adu\"\n idtype=[\"$adu_eis_idtype\"]\n uid= $(id -u $adu_user)\n" > "$eis_idservice_dir/$adu_eis_conf_file"
+                printf "[[principal]]\n name=\"deviceupdate\"\n idtype=[\"module\"]\n uid= $(id -u $adu_user)\n" > "$eis_idservice_dir/$adu_eis_conf_file"
```

Comment:
>deviceupdate [](start = 47, length = 12)

would it be better to have a define for this, e.g. `$adu_eis_name`?
see my other comment as ewell.